real_estate/models/estate_property.py

```diff
@@ -0,0 +1,179 @@
+from datetime import date, timedelta
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
```

Comment:
```suggestion
from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
```
Odoo imports should be seen as differently from normal py imports